### DISCUSSION TITLE
Does hyperspy support Brukers ESPRIT .rtx format?

### DISCUSSION THREAD

**Discussion by SaltyCam** [hyperspy/rosettasciio#253] [Q&A]
Hello,

I wanted to inquire does this have support for project files (.rtx) exported from Brukers ESPRIT software (EDS data, maps, linescans) ?

**Comment by ericpre** [comment]
No, it doesn't. The supported features of bruker files are documented in https://hyperspy.org/rosettasciio/user_guide/supported_formats/bruker.html.

**Comment by sem-geologist** [comment]
Code-wise it has some classes and functions already there which could allow to implement rtx quite easily directly, or functions and classes could be used externally. I will see how much things could be bent to make this more possible.
